src/main/java/com/keboola/tableexporter/CsvException.java

```diff
@@ -0,0 +1,7 @@
+package com.keboola.tableexporter;
+
+public class CsvException extends Exception {
```

Comment:
maybe we can put the exceptions to a separate namepsace so that it is more tidy?

Reply:
done